pkg/locodedb/utils.go

```diff
@@ -0,0 +1,124 @@
+package locodedb
+
+import (
+	"bytes"
+	"compress/gzip"
+	_ "embed"
+	"encoding/csv"
+	"io"
+	"strconv"
+	"sync"
+)
+
+var (
+	//go:embed data/countries.csv.gz
+	countriesData []byte
+
+	//go:embed data/locodes.csv.gz
+	locodesData []byte
+
+	locodeDataOnce sync.Once
+)
+
+func initLocodeData() (err error) {
+	locodeDataOnce.Do(func() {
+		mCountries, err = unpackCountriesData(countriesData)
+		if err != nil {
+			return
+		}
+		countriesData = nil
+		mLocodes, err = unpackLocodesData(locodesData)
+		locodesData = nil
+	})
+	return
+}
+
+type locodesCSV struct {
+	locationName string
+	subDivCode   string
+	subDivName   string
+	point        *Point
+	continent    *Continent
+}
+
+func unpackCountriesData(data []byte) (map[CountryCode]string, error) {
+	m := make(map[CountryCode]string)
+
+	gzReader, err := gzip.NewReader(bytes.NewReader(data))
+	if err != nil {
+		return m, err
+	}
+	defer gzReader.Close()
+
+	reader := csv.NewReader(gzReader)
```

Comment:
why not use our own `Table` and `ItarateAll` that are used for db generation generation?

Reply:
I am not really sure that we need to move table parsing functionality to pkg package

Reply:
why moving? we have our own rules for our own bases needed in both ways: for writing and reading. i do not see any problems when _pkg_ imports some _internals_ if it needs to reuse parsing rules